tools/InstallationTests/Common.ps1

```diff
@@ -193,14 +193,33 @@ function Test-Setup
 #############################
 function Test-Cleanup
 {
-     $global:ConfirmPreference = $global:oldConfirmPreference
-     $global:DebugPreference = $global:oldDebugPreference
-     $global:ErrorActionPreference = $global:oldErrorActionPreference
-     $global:FormatEnumerationLimit = $global:oldFormatEnumerationLimit
-     $global:ProgressPreference = $global:oldProgressPreference
-     $global:VerbosePreference = $global:oldVerbosePreference
-     $global:WarningPreference = $global:oldWarningPreference
-     $global:WhatIfPreference = $global:oldWhatIfPreference
+     #$global:ConfirmPreference = $global:oldConfirmPreference
+     #$global:DebugPreference = $global:oldDebugPreference
+     #$global:ErrorActionPreference = $global:oldErrorActionPreference
+     #$global:FormatEnumerationLimit = $global:oldFormatEnumerationLimit
+     #$global:ProgressPreference = $global:oldProgressPreference
+     #$global:VerbosePreference = $global:oldVerbosePreference
+     #$global:WarningPreference = $global:oldWarningPreference
+     #$global:WhatIfPreference = $global:oldWhatIfPreference
+
+    Delete-DownloadedCertAndPubSetting
+    
+    Remove-AllSubscriptions
+}
+
+Function Delete-DownloadedCertAndPubSetting
```

Comment:
Seems like this belongs in the same file as the code that downloads the cert and publishsettings